kolibri/plugins/learn/assets/src/views/LibraryPage/index.vue

```diff
@@ -49,63 +47,59 @@
           @setSidePanelMetadataContent="content => metadataSidePanelContent = content"
         />
         <!-- Other Libraires -->
-        <div v-if="!deviceId">
-          <KGrid gutter="12">
-            <KGridItem :layout="{ span: 6 }">
-              <span>
-                <h1>
-                  {{ $tr('otherLibraries') }}
-                </h1>
-              </span>
+        <div>
+          <KGrid>
+            <KGridItem
+              :layout12="{ span: 8 }"
+              :layout8="{ span: 4 }"
+              :layout4="{ span: 4 }"
+            >
+              <h1>
+                {{ $tr('otherLibraries') }}
+              </h1>
             </KGridItem>
-            <KGridItem :layout="{ span: 6, alignment: 'right' }">
-              <p
-                v-if="searching"
-                style="padding-top:20px"
-              >
-                {{ $tr('searchingOtherLibrary') }}
-                <KButton appearance="basic-link">
-                  {{ coreString('refresh') }}
-                </KButton>
-                <KIcon icon="disconnected" />
-              </p>
-              <p v-else>
+            <KGridItem
+              :layout12="{ span: 4 }"
+              :layout8="{ span: 4 }"
+              :layout4="{ span: 4 }"
+            >
+              <div v-if="searching" style="padding-top:20px">
+                <div class="sync-status">
+                  <div>
+                    {{ $tr('searchingOtherLibrary') }}
+                  </div>
+                  <div>
+                    <KButton appearance="basic-link">
+                      {{ $tr('refresh') }}
+                    </KButton>
+                  </div>
+                  <div>
+                    <KIcon
+                      icon="wifi"
+                    />
+                  </div>
+                </div>
+              </div>
+              <div v-else>
                 {{ coreString('viewMoreAction') }}
                 {{ $tr('pinned') }}
                 {{ $tr('showingAllLibraries') }}
                 {{ $tr('noOtherLibraries') }}
                 {{ $tr('searchingOtherLibrary') }}
-              </p>
+              </div>
             </KGridItem>
           </KGrid>
-          <PinnedNetworkResources />
         </div>
+        <PinnedNetworkResources
+          v-if="pinnedDevices"
+          :pinnedDevices="pinnedDevices"
+        />
         <!-- More  -->
-        <div v-if="!deviceId">
-          <h2>
-            {{ $tr('moreLibraries') }}
-          </h2>
+        <div>
+          <h2>{{ $tr('moreLibraries') }}</h2>
           <MoreNetworkDevices />
```

Comment:
I think applying the same concept as seen with `PinnedNetworkResources` would work. Creating an `unpinnedDevices`  `data` property and passing it into the `PinnedNetworkResources` as a prop would help with refreshing state of these devices(I think there is a refresh button somewhere). For now its okay for them to display the same devices. Later we can apply filtering based on results returned from the pinning API that we co-hacked with Jacob yesterday